src/tiled/preferences.h

```diff
@@ -88,6 +88,9 @@ class Preferences : public QObject
     Map::RenderOrder mapRenderOrder() const;
     void setMapRenderOrder(Map::RenderOrder mapRenderOrder);
 
+    qreal getTerrainEditorZoom(QString name);
```

Comment:
Naming style: getters do not have a "get" prefix, so this would be just `terrainEditorZoom`. Also, the getter should be defined `const`.

Efficiency: Please pass `QString` always as `const QString &`. Even if of minor impact, this avoids the reference counting so it's good to get used to always pass it like that.